pallets/task/src/mock.rs

```diff
@@ -24,10 +24,12 @@ frame_support::construct_runtime!(
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-		Task: pallet_task::{Pallet, Call, Storage, Event<T>},
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+		Did: pallet_did::{Pallet, Call, Storage, Event<T>},
```

Comment:
Why we need the DID and DAO pallet in the task mock?

Reply:
The `Dao` pallet is required to satisfy the `OrganizationId` verification during task creation/updates. The tests make use of organisation verification.

The `Did` pallet needs to be pulled in due to the tight coupling of `Dao` pallet to the `Did` pallet.